package.json

```diff
@@ -14,7 +14,8 @@
     "start": "next start -p $PORT",
     "test": "jest --testPathIgnorePatterns=e2e --runInBand",
     "test:e2e": "jest --testPathPattern=e2e --runInBand",
-    "test:e2e:local": "ts-node bin/e2e/prepare-e2e-tests.ts && ts-node bin/e2e/setting-sudo-ttl.ts && jest --testPathPattern=e2e --runInBand",
+    "test:e2e:local": "yarn test:e2e:delete-profile-user && yarn test:e2e:remove-screenshots && ts-node bin/e2e/prepare-e2e-tests.ts && ts-node bin/e2e/setting-sudo-ttl.ts && jest --testPathPattern=e2e --runInBand",
```

Comment:
The `deleteProfileUser` script is started each time we run the e2e tests in local.
It avoids us having to manually do it, either directly in the Profile's database, either with a call to the admin endpoint.

Let me know if you prefer to run it separately.

Reply:
I'm fine with this 👍.